lib/osm_community_index.rb

```diff
@@ -0,0 +1,26 @@
+module OsmCommunityIndex
+  def self.add_to_i18n
+    communities = Community.all
```

Comment:
Tricky question, but should we limit this using the same filter we use when choosing what to display? It would vastly limit how many translations we are pulling in...

Reply:
Fair point. I'm happy to limit them for now, and the filter can be expanded if/when we start using other communities elsewhere on the site.